docs/_book.yaml

```diff
@@ -97,7 +97,7 @@ upper_tabs:
       skip_translation: true
```

Comment:
While you're here, can you change the `name` from "API" to "Reference"

Reply:
Done